File: Estructura de datos/Ordenamiento de burbuja/BurbujaOrdenar.py
def Burbuja(arreglo):
    n = len(arreglo)
    for i in range(n-1):
        for j in range(n-i-1):
            if arreglo[j] > arreglo[j+1]:
                arreglo[j], arreglo[j+1] = arreglo[j+1], arreglo[j]

arreglo = [5, 2, 9, 1, 5, 6]
Burbuja(arreglo)
print(arreglo)

File: Estructura de datos/Ordenamiento de burbuja/BurbujaOrdenar.cpp
#include <iostream>
using namespace std;

void BurbujaOrdenar(int arreglo[], int n) {
    // Implementación del algoritmo BurbujaOrdenar
}

int main() {
    int arreglo[] = {5, 2, 9, 1, 5, 6};
    int n = sizeof(arreglo) / sizeof(arreglo[0]);
    BurbujaOrdenar(arreglo, n);
    for (int i = 0; i < n; i++) cout << arreglo[i] << " ";
}
